screen2.0/src/common/components/GenomeSwitch.tsx

```diff
@@ -72,7 +72,7 @@ const StyledSwitch = styled(Switch)(({ theme }) => ({
 }))
 
 const GenomeSwitch: React.FC<GenomeSwitchProps> = (props: GenomeSwitchProps) => {
-  const [checked, setChecked] = React.useState(false)
+  const [checked, setChecked] = React.useState(props.initialChecked)
```

Comment:
its better practice to import import React from "react" and import { useState } from "react" rather than using React.useState